### ISSUE TITLE
Domain request

### ISSUE THREAD

**Issue by Nazliozum** [rbind/support#305]
## Netlify website address

wizardly-montalcini-991c74.netlify.com

## Preferred rbind.io subdomain

nazliozumkafaee.rbind.io

### Agreement

- [x] By submitting this request, I promise I will at least write one blog post or create one web page on my website after I get the rbind.io subdomain.

**Comment by isteves** [comment]
@Nazliozum - we have just configured the rbind subdomain you requested. Please [set the rbind subdomain](https://support.rbind.io/about/) in your Netlify account.

![](https://user-images.githubusercontent.com/163582/31508235-11282aa2-af43-11e7-8204-3d019ef7fe33.png)

Note that there might be a hint "Check DNS configuration" in the domain section after adding the rbind subdomain -- it can be safely ignored.

Thanks!
Irene

**Comment by Nazliozum** [comment]
@isteves thank you very much! I have a question though. I added the custom domain, but it is not active. The site can be accessed still with the old domain but not the new one. Is there anything else I should do?

**Comment by Nazliozum** [comment]
Yes, that was the problem. Thank you for the reminder. The domain is active now.
